index.js

```diff
@@ -1,8 +1,67 @@
+}
+
 
 // Iteration 3: Loops
 
+let driver = ""
+for (let i = 0; i < Hacker1.length; i++){ 
+    driver = driver + Hacker1[i] + " ";
+
+}
+console.log(driver.toUpperCase());
+
+
+let navigator = ""
+for (let f = Hacker2.length-1; f >= 0; f--){
+    navigator = navigator + Hacker2[f];
+}
+
+console.log(navigator);
+
+
+
+if (Hacker2.localeCompare(Hacker1) === 1) {
+    console.log("The driver's name goes first.")
+}else if (Hacker1.localeCompare(Hacker2) === 1) {
+    console.log("Yo, the navigator goes first definitely.")
+}
+else {
+    console.log("What?! You both have the same name?")
+}
+
+//BONUS TIME 1 (ongoing)
+
+let longText = `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean eu turpis tempor, viverra leo et, condimentum velit. Proin eget lacus eu eros feugiat accumsan. Duis rutrum magna eget auctor tempus. Integer imperdiet, neque at euismod venenatis, nunc dolor dapibus elit, ut luctus libero risus vestibulum nisi. Phasellus viverra diam tortor, vel pretium mi vestibulum eu. Vestibulum augue lectus, maximus vitae consectetur vel, interdum eu tortor. Vestibulum enim leo, tempus id quam ut, fringilla sodales odio. Fusce odio ligula, gravida vel purus et, accumsan vestibulum tortor. Fusce finibus eget urna sed tincidunt. Sed eget odio in mauris eleifend consectetur vulputate quis lacus.
+Praesent dictum tellus in porta pharetra. Etiam pharetra sed lorem sed consectetur. Nullam vel metus hendrerit, varius velit non, pharetra magna. Sed euismod felis quam, a maximus ex finibus quis. Morbi lacinia tortor quis odio iaculis vulputate. Fusce imperdiet fringilla arcu, nec pharetra magna. Etiam efficitur urna sed sapien commodo consectetur. Quisque egestas dolor tortor, vitae aliquet nunc sagittis vel. Pellentesque ut lacus rhoncus, lacinia ligula vel, porta nibh. Vestibulum in aliquam tortor. Donec eu eleifend velit. Pellentesque porttitor sodales elit, sed hendrerit ipsum pulvinar et. Quisque non posuere justo. Curabitur suscipit arcu eget turpis scelerisque, sit amet egestas lectus pharetra. Vestibulum magna lectus, sagittis sit amet velit interdum, mattis pellentesque libero.
+Etiam massa enim, viverra ac quam quis, consectetur ornare ex. Morbi finibus laoreet lorem et consequat. Curabitur sed gravida neque. Praesent gravida non nisl ac tempus. Sed scelerisque arcu faucibus, sollicitudin libero a, placerat libero. Sed libero leo, tincidunt sit amet mauris id, scelerisque eleifend est. Quisque magna lorem, commodo a nulla ac, aliquam ullamcorper ante. Integer pharetra sit amet dolor ac pharetra. Pellentesque ornare ullamcorper luctus. Nam sodales enim id lectus consectetur, vel tempus arcu scelerisque. Morbi tortor nibh, pulvinar ut dui finibus, volutpat blandit tellus. Nam tincidunt ex metus. Aenean mollis ipsum nec tincidunt interdum. Pellentesque consectetur elit nulla, id mattis massa convallis sodales.`
+
+
+//wordCount = longText.length-1;
+//console.log(wordCount);
+console.log(longText.length);
```

Comment:
`.length` will return you the amount of characters in `longText` and not the ammount of words. So one way of counting the words is counting the number of spaces between words (similar to the for loop that you have below)